common/class_attributes/dump_mode.md

```diff
@@ -0,0 +1,1260 @@
+---
+layout: default
+grand_parent: Common
+parent: Class Attributes
+---
+
+# Dump Mode
+
+What is Dump Mode and how to use it.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .note}
+> _Changes made using the functionality described here are scoped to the target "Admin" object instance._
+
+&nbsp;
+
+{: .note}
+> _See [Dump Processing](../../misc/dump_processing) for more information about **Dump Processing**._
+
+&nbsp;
+
+{: .warning}
+> _**Dump Mode** should only be used for **Development** and **Debugging** purposes._
+
+&nbsp;
+
+**Dump Mode** can be enabled on any "Admin" object by setting the `dump_mode` class attribute to `True` through the constructor. Once enabled, a dump file containing the **Raw Result** returned by **IRRSMO00** will be created at `~/.pyracf/dump` with the naming convention `pyracf.<timestamp>.<md5>.dump` on every security request *(Both successes and failures)*.
+
+## Example
+
+###### Python Script
+```python
+from pyracf import UserAdmin
+user_admin = UserAdmin(dump_mode=True)
+
+user_admin.extract("squidwrd")
+```
+
+###### Console Output
+```console
+[ INFO ] Raw security result XML has been written to '/u/testuser/.pyracf/dump/pyracf.<timestamp>.<md5>.dump'.
+```
+
+{: .note }
+> _If both **Dump Mode** and [Debug Logging](../debug) are enabled, a **Hex Dump** of the **Raw Result XML** returned by **IRRSMO00** will also be printed to the console. Also keep in mind that secrets are redacted from the **Raw Security Result XML Hex Dump** that is printed to the console, which is done for everything that may contain sensitive information that pyRACF prints to the console._

```

Comment:
How about

`Also keep in mind that secrets are redacted from all output that pyRACF prints to the console. This includes the **Raw Security Result XML Hex Dump** generated by this combination of features.`